laplace/curvature/backpack.py

```diff
@@ -41,7 +41,7 @@ def jacobians(self, x):
                 if model.output_size > 1:
                     out[:, i].sum().backward()
                 else:
-                    out.sum().backward()
+                    out.sum().backward(retain_graph=True)
                 to_cat = []
                 for param in model.parameters():
                     to_cat.append(param.grad_batch.detach().reshape(x.shape[0], -1))
```

Comment:
This `detach()` is fine?

Reply:
Fixed in [bc50025](https://github.com/AlexImmer/Laplace/pull/120/commits/bc50025d079abae01a65ed1dce8c3e3fe0975e6b)